README.md

```diff
@@ -39,11 +39,18 @@ bruhl:
 2. Check the ruby version in `.ruby-version` and ensure you have it installed locally e.g. `rbenv install 2.5.1`
 3. Install [bundler](https://bundler.io/) (the latest Heroku-compatible version): `gem install bundler`
 4. [Install Postgres](https://github.com/codeforamerica/howto/blob/master/PostgreSQL.md). If setting up Postgres.app, you will also need to add the binary to your path. e.g. Add to your `~/.bashrc`:
-`export PATH="$PATH:/Applications/Postgres.app/Contents/Versions/latest/bin"`
+`export PATH="$PATH:/Applications/Postgres.app/Contents/Versions/latest/bin"`.
+5. [Install PostGIS](https://postgis.net/install/), the Postgres geospatial extension, if it's not included in your distribution. Postgres.app comes with postgis.
 
 ### Application Setup
 
 1. Install ruby gem dependencies: `bundle install`
 2. Create the databases and load schema and seeds: `bin/rails db:setup`
 3. Run the tests: `bin/rspec`
 4. Run the server: `bin/rails server`, and visit the web-browser: [`http://localhost:3000`](http://localhost:3000)
+5. Load cities: `rake cities:load`
+6. Load service requests: `rake cities:service_requests`
```

Comment:
These lines should be unnecessary when running `db:setup` so far as it will load cities and generate _fake_ service requests. Perhaps the better improvement would be to make a section in the readme that differentiates between “developing with fake data” and “running it for reals”